.gitignore

```diff
@@ -61,5 +61,9 @@ chaindata/
 genesis.json
 keystore/
 
+# Test remnants
+logs/
+migrations/
```

Comment:
Cool.  Then I likely had a different rule than this and my memory is failing me.  Thanks for proving me wrong :P